bin/sapconf

```diff
@@ -92,8 +93,7 @@ stop() {
       log "no active sapconf tuning - profile=$profile -, so nothing to revert"
       exit 0
     else
-      echo "$profile" > /var/lib/sapconf/last_profile
```

Comment:
please don't remove this line. During the stop of sapconf (may be because of a normal reboot) we need to store the current running profile for the next start of sapconf. This is the only way for sapconf to know after a reboot, which profile was active before.

Reply:
This is sensce less. It will be written already by start.

Reply:
yes, your right. And with the changes for the 'profile'-specific parts below we should have covered all possible cases.